spec/spec_helper.rb

```diff
@@ -36,6 +36,7 @@
   config.include GovukSchemas::RSpecMatchers
   config.include AuthenticationHelper, type: ->(spec) { spec.in?(%i[feature request]) }
   config.include BulkDataHelper
+  config.include Capybara::RSpecMatchers, type: :request
```

Comment:
Ah yes - I had tried to avoid adding this as much as possible so I'd used include before it was added. have_content does seem a more idiomatic option.